cmd/kubelet/app/server.go

```diff
@@ -1009,15 +1008,8 @@ func InitializeTLS(kf *options.KubeletFlags, kc *kubeletconfiginternal.KubeletCo
 		return nil, err
 	}
 
-	if len(tlsCipherSuites) > 0 {
-		insecureCiphers := flag.InsecureTLSCiphers()
-		for i := 0; i < len(tlsCipherSuites); i++ {
-			for cipherName, cipherID := range insecureCiphers {
-				if tlsCipherSuites[i] == cipherID {
-					klog.Warningf("Use of insecure cipher '%s' detected.", cipherName)
-				}
-			}
-		}
+	for _, cipherName := range cliflag.InsecureTLSCipherNamesUsed(kc.TLSCipherSuites) {
```

Comment:
it may be even more optimal to return insecure certs list from the `cliflag.TLSCipherSuites`. Eliminating back mapping from IDs to names here is a good thing fro sure